metron-analytics/metron-profiler-client/src/test/java/org/apache/metron/profiler/client/GetProfileTest.java

```diff
@@ -116,6 +119,33 @@ public void setup() {
                     .build());
   }
 
+  private Context setup2() {
```

Comment:
What does `setup2` do differently than `setup`?  Could we comment or rename the function to be more descriptive?